.github/workflows/sign-and-release-to-github.yml

```diff
@@ -115,7 +116,7 @@ jobs:
 
       - uses: actions/upload-artifact@v2
         with:
-          name: fx-private-relay-stage-chromium.zip
+          name: fx-private-relay-extension.zip
```

Comment:
I renamed this since AFAIK this same zip can be uploaded to both the Chrome web store and AMO.